tests/modules/programs/foot/empty-settings.nix

```diff
@@ -0,0 +1,16 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.foot.enable = true;
+
+    nixpkgs.overlays =
+      [ (self: super: { alacritty = pkgs.writeScriptBin "dummy-foot" ""; }) ];
```

Comment:
```suggestion
      [ (self: super: { foot = pkgs.writeScriptBin "dummy-foot" ""; }) ];
```

Reply:
Yes as you can see my implementation is based on the alacritty module. 😅